images-list

```diff
@@ -124,8 +129,6 @@ quay.io/kiali/kiali rancher/kiali-kiali v1.28.0
 quay.io/kiali/kiali rancher/kiali-kiali v1.29.0
 quay.io/prometheus-operator/prometheus-config-reloader rancher/coreos-prometheus-config-reloader v0.43.0
 quay.io/prometheus-operator/prometheus-operator rancher/coreos-prometheus-operator v0.43.0
-quay.io/prometheus/prometheus rancher/prom-prometheus v2.19.2
-quay.io/prometheus/prometheus rancher/prom-prometheus v2.22.0
```

Comment:
Why did these get deleted ? Nothing should get deleted from this file.

Reply:
hmmm bad rebase? Fixed!